conversions.js

```diff
@@ -902,7 +880,7 @@ convert.gray.hex = function (gray) {
 	return '000000'.slice(string.length) + string;
 };
 
-convert.rgb.gray = function (rgb) {
+export const rgbToGray = rgb => {
```

Comment:
In order for this change to not be breaking (since the library just majored), these functions would still need to be attached to the default export (and possibly deprecate the default export with a recommendation to use the name exports?)

Reply:
I don't think so. This will bloate the package and increase the maintenance cost.

This is fine to create another breaking change. Users can always choose which version they want.